pages/faq.md

```diff
@@ -1,57 +1,47 @@
 
-Vanaf 2015 heeft een samenwerking tussen de Open State Foundation, VNG Realisatie, Argu en Ontola het platform openraadsinformatie.nl (nu: openbesluitvorming.nl) opgeleverd. Dit platform richtte zich op het ophalen en vervolgens doorzoekbaar maken van zoveel mogelijk digitaal gepubliceerde politieke en bestuurlijke besluitvormingsstukken van gemeenten en dit op herbruikbare wijze als open data te ontsluiten. Sinds de intrede van de Wet open overheid in 2022 is de scope uitgebreid naar alle Nederlandse politieke en bestuurlijke overheidsorganen. 
+# Waarom is ORI-A een XML-schema?
+ORI-A is als XML Schema Document (XSD) ontworpen, zodat het gezamenlijk met [MDTO](https://www.nationaalarchief.nl/archiveren/mdto) kan worden gebruikt bij migraties naar het e-depot. Metagegevens over informatieobjecten en bestanden kan dan in MDTO worden uitgedrukt, en metagegevens over raadsinformatie in ORI-A.
 
-De ontwikkeling van openbesluitvorming.nl splitste zich uiteen in het ontwerp van een pull-API ([Open Raadsinformatie API](https://github.com/openstate/open-raadsinformatie)), een elastic search front-end voor het doorzoekbaar maken van deze data ([openbesluitvorming](https://github.com/ontola/openbesluitvorming)) en een informatiemodel voor raadsinformatie op basis waarvan een Open API Specificatie wordt opgesteld ([ODS Open Raadsinformatie](https://github.com/VNG-Realisatie/ODS-Open-Raadsinformatie)). Deze laatste heeft als basis gediend voor ORI-A.
+# Waarom heeft ORI-A geen aggregatieniveaus?
+ORI-A draait om de entiteit vergadering. ORI-A is zo ontworpen, dat een ORI-A XML altijd over maximaal één vergadering informatie bevat. Het is wel mogelijk om informatie over één vergadering in meerdere ORI-A.xml-bestanden te verspreiden. 
 
-Het ORI-A XML-schema is gebaseerd op het informatiemodel onder de Open API Specificatie. De elementen, attributen en onderlinge relaties in het informatiemodel zijn zoveel mogelijk één-op-één overgenomen. De gedachte hierachter is dat we een organisatie die al werkt met ORI zo min mogelijk meerwerk willen bezorgen om ook ORI-A te implementeren. ORI-A volgt de doorontwikkeling van ORI en zal waar nodig worden aangepast om gezamenlijke implementatie zo laagdrempelig mogelijk te maken.
+In de praktijk zal ORI-A vaak gezamenlijk met MDTO worden ingezet. Het is dan mogelijk om je raadsinformatie per vergadering in één ORI-A.xml op te nemen en je informatieobjecten in een MDTO [sidecar-structuur](https://www.nationaalarchief.nl/archiveren/mdto/specificatie-submission-information-package/structuur#:~:text=Sidecar%2D%20structuur&text=Een%20bestand%20kan%20op%20elk,heeft%20zijn%20eigen%20MDTO%20metagegevensbestand.). Je kan de ORI-A.xml ook opsplitsen en opnemen als [aanvullende metagegevens](https://www.nationaalarchief.nl/archiveren/mdto/aanvullendeMetagegevens) bij je informatieobjecten.  
```

Comment:
> https://www.nationaalarchief.nl/archiveren/mdto/specificatie-submission-information-package/structuur#:~:text=Sidecar%2D%20structuur&text=Een%20bestand%20kan%20op%20elk,heeft%20zijn%20eigen%20MDTO%20metagegevensbestand.

Ik denk dat deze kortere URL volstaat: https://www.nationaalarchief.nl/archiveren/mdto/specificatie-submission-information-package/structuur

Reply:
~~in een MDTO [sidecar-structuur]~~ in een [MDTO sidecar-structuur] 

Geeft iets meer context over de link. Maar je zou ook kunnen beargumenteren dat de eerste variant correct is, iig wanneer de pagina waar je naar toe linkt een super algemene uitleg van side-car structuren geeft